src/components/date-picker/DatePicker.tsx

```diff
@@ -562,9 +561,9 @@ class DatePicker extends Component<
             className={classNames({
               active: showPicker,
             })}
-            iconElement={
+            rightDecorators={
               <Icon
-                className={classNames({
+                className={classNames('tk-input__right-decorators__clickable', {
```

Comment:
are there non clickable decorators? and is this documented?

Reply:
The developer is free to put whatever he wants in the rightDecorators prop, so yes, it can be things not clickable

So if he wants a clickable decorator, then we propose tk-input__right-decorators__clickable

And yes it is documented :)
![image](https://user-images.githubusercontent.com/56824367/146236323-fa12fce0-291d-4514-88c7-07da0c997bce.png)